src/shared/sam/debugger/awsSamDebugger.ts

```diff
@@ -642,7 +711,7 @@ export class SamDebugConfigProvider implements vscode.DebugConfigurationProvider
                 return await javaDebug.invokeJavaLambda(this.ctx, config)
             }
             default: {
-                throw Error(`unknown runtimeFamily: ${config.runtimeFamily}`)
+                throw new Error(`unknown runtimeFamily: ${config.runtimeFamily}`)
```

Comment:
is there a technical difference here? should we have a lint rule for it?

Reply:
For this class implementation in particular you will get the same result but calling a constructor function without using `new` is functionally different. The `Error` constructor is likely using `new` internally if you don't call it with the `new` keyword so you get the same thing anyway. 

Wouldn't be a bad idea to lint for `new` preceding any Error ctor call if only for consistency.